packages/site/cms-site/src/blocks/DamFileDownloadLinkBlock.tsx

```diff
@@ -38,3 +39,5 @@ export const DamFileDownloadLinkBlock = withPreview(
     },
     { label: "DamFileDownloadLink" },
 );
+
+export const convertToFileSize = (fileSize: number | string) => filesize(fileSize);
```

Comment:
Do we really need this if we do nothing than use a third party library? Why not use the third party library in Demo directly?

Reply:
The idea was to provide a helper from comet that can be used when creating a component for the site

Reply:
@thomasdax98 what do you think? I don't think it makes sense to add this helper.

Reply:
I also don't see a benefit. Please remove it for now @fichtnerma

Reply:
done